healthworkforce/docker/importer/volume/endpoint.json

```
@@ -0,0 +1,103 @@
+{
+  "name": "mCSD Orchestration",
+  "endpoint": {
+    "pattern": "/mcsd"
+  },
+  "transformation": {
+    "input": "JSON",
+    "output": "JSON"
+  },
+  "requests": {
+    "lookup": [
+      {
+        "id": "gofr-location",
+        "config": {
+          "method": "get",
+          "url": "http://mock-services:4000/gofr-location-mock/_history",
+          "headers": {
+            "Content-Type": "application/json"
+          },
+          "params": {
+            "_since": {
+              "path": "state.system.timestamps.lookupRequests.gofr-location.requestStart",
+              "prefix": null,
+              "postfix": null  
```

There are trailing blank spaces on all of the postfix fields.

```suggestion
              "postfix": null
```